guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,173 @@
+[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+= Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+
+Use this procedure to upgrade your {Project} installation from Enterprise Linux 7 to Enterprise Linux 8.
+LEAPP is the supported tool for Enterprise Linux 7 to Enterprise Linux 8 upgrades.
+
+.Prerequisites
+* {Project} {ProjectVersionPrevious} running on CentOS Linux 7.
```

Comment:
Maybe it's best to say "Enterprise Linux 7" again?

If this procedure is used for both Foreman and Satellite, then maybe your can also use an attribute to differentiate between RHEL and Enterprise Linux 7. Alternatively, we can discuss setting `RHEL` to CentOS for Foreman build targets?